play_motion:
  motions:
    look_down:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [0.0, -0.2]
        time_from_start: 0.0
    look_left:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [0.5, -0.25]
        time_from_start: 0.0 
    look_down_left:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [0.5, -0.55]
        time_from_start: 0.0
    look_right:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [-0.5, -0.25]
        time_from_start: 0.0 
    look_down_right:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [-0.5, -0.55]
        time_from_start: 0.0
    check_table:
      joints: [torso_lift_joint, head_1_joint, head_2_joint]
      points:
      - positions: [0.25, 0.0, -0.25]
        time_from_start: 0.0
    check_annoying_table:
      joints: [torso_lift_joint, head_1_joint, head_2_joint]
      points:
      - positions: [0.05, 0.0, -0.25]
        time_from_start: 0.0
    check_annoying_table_low:
      joints: [torso_lift_joint, head_1_joint, head_2_joint]
      points:
      - positions: [0.05, 0.0, -0.55]
        time_from_start: 0.0
    check_table_low:
      joints: [torso_lift_joint, head_1_joint, head_2_joint]
      points:
      - positions: [0.25, 0.0, -0.55]
        time_from_start: 0.0 
    back_to_default:
      joints: [torso_lift_joint, head_1_joint, head_2_joint]
      points:
      - positions: [0.15, 0.0, 0.0]
        time_from_start: 0.0
    load_unload:
      joints: [torso_lift_joint]
      points:
      - positions: [0.05]
        time_from_start: 0.0
    tablet:
      joints: [torso_lift_joint, head_1_joint, head_2_joint]
      points:
      - positions: [0.05, 0.0, -0.5]
        time_from_start: 0.0
